### ISSUE TITLE
Ribbon scrolling on touch screen

### ISSUE THREAD

**Issue by AelyPro** [fluentribbon/Fluent.Ribbon#1176]
Hello,
I have trouble getting the scrolling on a ribbon right.
At the top of my application I have a ribbon with multiple button. 
Once the window get too small I can scroll throughthe button with the 2 arrow at each side and with the mouse.
As my application is mostly used on a touch screen I'd like my users to be able to scroll through the menu simply by dragging their finger on it.

![ribbonScrolling](https://github.com/fluentribbon/Fluent.Ribbon/assets/62246276/9f4e00ec-33dd-43ff-ae17-4e6db404e1c6)

My code is as follow

```
<Fluent:Ribbon AreTabHeadersVisible="False">
   <Fluent:Ribbon.ToolBarItems>
      <Fluent:Button
         Command="{Binding AboutCMD}"
         Icon="{StaticResource aboutIcon}"
         LargeIcon="{StaticResource aboutIcon}"
         Size="Middle"
         ToolTip="À-Propos" />
   </Fluent:Ribbon.ToolBarItems>

   <Fluent:RibbonTabItem Header="">
      <Fluent:RibbonGroupBox>
         <!--  Refresh  -->
...        
      </Fluent:RibbonGroupBox>
   </Fluent:RibbonTabItem>
</Fluent:Ribbon>
```

---
### Environment

- Fluent.Ribbon __v?.?.?__
- Windows 10 22H2
- .NET Framework net7.0-windows

**Comment by batzen** [comment]
Using one finger doesn't trigger the scrolling.
Your users have to use two fingers to trigger scrolling.
Or doesn't it work when they are using two fingers?
It should work as every scroll viewer works when using touch on Windows.

**Comment by AelyPro** [comment]
It doesn't seem to work. even with 2 fingers. 
Is it because this is a horizontal scroll ?

**Comment by batzen** [comment]
It's a horizontal scroll, so you also have to use touch in that direction.

**Comment by AelyPro** [comment]
Yep, I tried scrolling with 1,2 and 3 finger in all the directions and none worked.
Scrolling with the mouse wheel is fine though.

**Comment by batzen** [comment]
That's strange.
I tested touch scrolling using my phone and a program that forwards touch events to my PC, as i don't have a touch display to do a real world test.

Will try to test that again and have a closer look at the types of events i get from the program to make sure the program does not translate the touch events to mouse scroll events.
